src/main/flight/pid.c

```diff
@@ -253,21 +253,77 @@ void pidInitConfig(const pidProfile_t *pidProfile) {
     relaxFactor = 1.0f / (pidProfile->setpointRelaxRatio / 100.0f);
     levelGain = pidProfile->P8[PIDLEVEL] / 10.0f;
     horizonGain = pidProfile->I8[PIDLEVEL] / 10.0f;
-    horizonTransition = 100.0f / pidProfile->D8[PIDLEVEL];
+    horizonTransition = (float)pidProfile->D8[PIDLEVEL];
```

Comment:
I took a long look at changing it back to "horizonTransition = 100.0f / pidProfile->D8[PIDLEVEL]".  In the horizon_tilt_mode=EXPERT case, it can be reworked to save a division.  However, in the horizon_tilt_mode=SAFE case, the 'inclinationLevelRatio' value is applied to 'sensitFact' before the "100/sensitFact" happens, so I don't see a way to save a division there.  (There would need to be a "1.0/inclinationLevelRatio" operation.)  So, I'm not seeing a way to change it that doesn't get complicated.

(BTW there's no "d_level=0" check in the current code; it will cause a division-by-zero error.)